src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randomColor = new ColorSupplier();
+        Ball randomlyBall = new Ball();
+        Random random = new Random();
+        for (int i = 0; i < random.nextInt(100); i++) {
```

Comment:
100 should be a constant field

also, we can declare new ColorSupplier() and new Random() as fields. so that we don't create a new object each time the method is called